errors

```diff
@@ -0,0 +1,5 @@
+[failed to validate configuration ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml: Failed resolve MultiStageTestConfiguration test/gcp-ipi-sdn-migration-ovn-p2-f7: workflow/cucushift-installer-rehearse-gcp-ipi-sdn: parameter \"TEST_ADDITIONAL\" is overridden in [test/gcp-ipi-sdn-migration-ovn-p2-f7] but not declared in any step
```

Comment:
@memodi for test gcp-ipi-sdn-migration-ovn-p2-f7 which workflow is not for golang, you do not need to add TEST_ADDITIONAL for it. if you want it, you need to change chain cucushift-sdn-migration-ovn (but it seems the chain's target is not to run any golang and cucu case.)

same for gcp-ipi-sdn-migration-ovn-p2-f14.

Reply:
@kuiwang02 - I removed migration tests in commit [1b0f16d](https://github.com/openshift/release/pull/35758/commits/1b0f16daa9c47b0eac66f0f400aa61cbf4796ab8) ,I don't need to enable for them, the errors are gone now.

Reply:
@memodi it is good. please remove this errors file from this PR. thanks

Reply:
ah, I missed that, deleted it. thanks!